.github/actions/linux/build-libmongocrypt/action.yml

```diff
@@ -0,0 +1,29 @@
+name: "Build libmongocrypt"
+description: "Installs libmongocrypt"
+inputs:
+  version:
+    description: "Libmongocrypt version to install (major.minor)"
+    required: true
+runs:
+  using: composite
+
+  steps:
+    - name: Add repository key
+      shell: bash
+      run: sudo sh -c 'curl -s --location https://pgp.mongodb.com/libmongocrypt.asc | gpg --dearmor >/etc/apt/trusted.gpg.d/libmongocrypt.gpg'
+
+    - name: Add repository
```

Comment:
Is there a particular reason that libmongocrypt is installed as a package, while libmongoc is compiled for source?

Wouldn't the approach here assume that the build is running on an Ubuntu or Debian OS? I suppose that's OK since `runs-on: "ubuntu-latest"` is specified for the `test-system-libs` workflow.